src/main/resources/com/atlassian/bitbucket/jenkins/internal/scm/SelectBranchTrait/help.properties

```diff
@@ -0,0 +1 @@
+select.branch.trait.help=Jenkins will only build branches that are the source branches of open pull requests. 
```

Comment:
My main concern here is that this feature is explicitly and only for our plugin, but it appears in a group of general behaviours- it is only demarcated as belonging to our plugin in the "from" field of the help dialogue (which I think some customers may miss- I certainly have before.)
I think we need the help text to clearly state this feature is for Bitbucket Server only, unless we can find a way for the behaviour to only appear in Bitbucket SCM or Bitbucket SCM source behaviour menus.